api/swagger-spec/v1.json

```diff
@@ -601,7 +601,7 @@
       ]
      },
      {
-      "type": "v1.Status",
```

Comment:
No I dont think so. Since the clients send and retrieve the json equivalent.
cc @abonas and @madhuri-rai07 to confirm (and heads up)